sensor/tests/resource/helper.go

```diff
@@ -295,6 +295,32 @@ func (c *TestContext) LastResourceStateWithTimeout(matchResourceFn MatchResource
 	}
 }
 
+// WaitForDeploymentEvent waits until sensor process a given deployment
+func (c *TestContext) WaitForDeploymentEvent(name string) {
+	c.WaitForDeploymentEventWithTimeout(name, 3*time.Second)
+}
+
+// WaitForDeploymentEventWithTimeout waits until sensor process a given deployment
+func (c *TestContext) WaitForDeploymentEventWithTimeout(name string, timeout time.Duration) {
+	timer := time.NewTimer(timeout)
+	ticker := time.NewTicker(500 * time.Millisecond)
+	lastErr := errors.Errorf("the deployment %s was not processed", name)
```

Comment:
Technically it could've been processed, it was just never sent by Sensor.
```suggestion
	lastErr := errors.Errorf("the deployment %s was not sent" name)
```

Reply:
Done